doc.json

```diff
@@ -342,6 +342,88 @@
         }
       }
     },
+    "/followuser/{id}": {
+      "get": {
+        "tags": [
+          "Follow"
+        ],
+        "summary": "Follow user",
+        "description": "Follow a user",
+        "consumes": [
+          "application/x-www-form-urlencoded",
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "x-access-token",
+            "in": "header",
+            "description": "A token to verify the user",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "ID of user to follow",
+            "required": true,
+            "type": "integer",
+            "format": "int64"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "User followed successfully."
+          },
+          "403": {
+            "description": "You cannot follow a user twice."
+          }
+        }
+      }
+    },
+    "/unfollowuser/{id}": {
```

Comment:
@youngestdj `follow` and `unfollow` should suffice except we plan to implement some other `follow` feature in the future.

Reply:
Route names have been changed